scripts/install/lidarr.sh

```diff
@@ -8,26 +9,56 @@
 #   changes/dates in source files. Any modifications to our software
 #   including (via compiler) GPL-licensed code must also be made available
 #   under the GPL along with build & install instructions.
-user=$(cut -d: -f1 < /root/.master.info)
-ip=$(ip route get 1 | sed -n 's/^.*src \([0-9.]*\) .*$/\1/p')
-distribution=$(lsb_release -is)
-version=$(lsb_release -cs)
-#shellcheck source=sources/functions/mono
-. /etc/swizzin/sources/functions/mono
-mono_repo_setup
-apt_install libmono-cil-dev libchromaprint-tools
-
-echo_progress_start "Downloading Lidarr release and extracting"
-cd /home/${user}/
-wget -O lidarr.tar.gz -q $(curl -s https://api.github.com/repos/Lidarr/Lidarr/releases | grep linux.tar.gz | grep browser_download_url | head -1 | cut -d \" -f 4)
-tar xf lidarr.tar.gz
-rm -rf lidarr.tar.gz
-chown -R ${user}: /home/${user}/Lidarr
-echo_progress_done "Lidarr downloaded and extracted"
-
-echo_progress_start "Configuring Lidarr"
-if [[ ! -d /home/${user}/.config/Lidarr/ ]]; then mkdir -p /home/${user}/.config/Lidarr/; fi
-cat > /home/${user}/.config/Lidarr/config.xml << LID
+
+if [ -z "$LIDARR_OWNER" ]; then
+    if ! LIDARR_OWNER="$(swizdb get lidarr/owner)"; then
+        LIDARR_OWNER=$(_get_master_username)
+        echo_info "Setting Lidarr owner = $LIDARR_OWNER"
+        swizdb set "lidarr/owner" "$LIDARR_OWNER"
+    fi
+else
+    echo_info "Setting Lidarr owner = $LIDARR_OWNER"
+    swizdb set "lidarr/owner" "$LIDARR_OWNER"
+fi
+
+install() {
+    user="$LIDARR_OWNER"
+    apt_install mediainfo sqlite3 libchromaprint-tools
+
+    echo_progress_start "Downloading source files"
```

Comment:
we're not compiling from source; this should be changed to Downloading Release or ~

Reply:
```suggestion
    echo_progress_start "Downloading release archive"
```